devel/pkgconf/Portfile

```diff
@@ -1,9 +1,10 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem                  1.0
+PortGroup                   legacysupport 1.1
```

Comment:
you should specify which macOS version needs it and for what missing function as we usually do.

Reply:
Done in latest commit